BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift

```diff
@@ -138,29 +138,36 @@ final class AppCoordinator: Coordinator {
     }
     
     func processExternalDocument(withUrl url: URL, sourceApplication: String?) {
-        
+        var doc: GiniCaptureDocument?
```

Comment:
[Suggestion] This is used only inside the `documentBuilder.build() {}` block. I would suggest to move it there to reduce the distance between declaration and usage.